tasmota/tasmota_xsns_sensor/xsns_90_hrg15.ino

```diff
@@ -57,8 +59,12 @@ struct RG15 {
   float rate = NAN;
   uint16_t time = RG15_EVENT_TIMEOUT;
   uint8_t init_step;
+  uint32_t dataReceived = 0;
+  bool isSensorAvailable = false;
 } Rg15;
 
+#define BOOL_TO_STRING(b) ((b) ? "true" : "false")
```

Comment:
Do not use true/false. Use 1/0 as this can be handled by less code and any Home Automation tool.